python/tvm/relax/frontend/onnx_frontend.py

```diff
@@ -1472,7 +1471,25 @@ def from_onnx(
                 param_list = [v for k, v in self._inputs.items() if isinstance(v, relax.Var)]
                 output_var = self.bb.emit_output(outputs)
             self.bb.emit_func_output(output_var, params=param_list)
-        return self.bb.get()
+        relax_mod = self.bb.get()
+        self._check_span_info_is_attached(relax_mod)
+        return relax_mod
+
+    def _check_span_info_is_attached(self, relax_mod: IRModule):
+        """Checks that span information is attached to all Relax call nodes."""
+
+        @relax.expr_functor.visitor
+        class SpanValidator(PyExprVisitor):
```

Comment:
I think it makes more sense to do this check as part of testing. It's likely not a good idea to error out an import if some calls dont have spans attached since many use cases wouldnt rely on them, this would make any span related bugs effectively completely disable the onnx importer. Having it in the tests instead should check that behavior is appropriate without impacting users.